lib/pinga-server/src/config.rs

```diff
@@ -1,18 +1,16 @@
-use std::{env, path::Path};
-
 use buck2_resources::Buck2Resources;
 use derive_builder::Builder;
 use serde::{Deserialize, Serialize};
 use si_crypto::{CryptoConfig, SymmetricCryptoServiceConfig, SymmetricCryptoServiceConfigFile};
 use si_data_nats::NatsConfig;
 use si_data_pg::PgPoolConfig;
 use si_std::CanonicalFileError;
+use std::{env, path::Path};
 use telemetry::prelude::*;
 use thiserror::Error;
+use ulid::Ulid;
 
-pub use si_crypto::CycloneKeyPair;
```

Comment:
Why is this export removed?

Reply:
Clippy seems to believe the import is unused. Does not mean that we don't want to keep though, but I see I've seen it on `main` for a couple weeks now.
```
warning: unused import: `si_crypto::CycloneKeyPair
```